_capabilities/fault_lines_UCLA

```diff
@@ -0,0 +1,53 @@
+---
+title: "Fault-Line Image Explanations"
+excerpt: "Algorithm for generating conceptual and counterfactual explanations for an image classification model."
+tags:
+  - Analytics
+  - Computer Vision
+  - Saliency
+  - Explanation Framework
+   
+submission_details:
+  resources: 
+    papers:
+      - https://ojs.aaai.org/index.php/AAAI/article/view/5643
+    software:
+      - https://github.com/arjunakula/faultline_explainer
+      - https://github.com/arjunakula/CoCoX
+   
+  # Optional information describing artifact
+  version: 1.0
+  size: 100MB
+  license: NA
```

Comment:
You should consider creating a license for the Github repo as that may allow more users to use your software.

Reply:
done